pkgs/tools/security/tpm2-abrmd/default.nix

```
@@ -1,24 +1,36 @@
 { stdenv, fetchurl, lib
-, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
+, makeWrapper, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
 
 stdenv.mkDerivation rec {
   pname = "tpm2-abrmd";
-  version = "2.2.0";
+  version = "2.3.2";
 
   src = fetchurl {
     url = "https://github.com/tpm2-software/${pname}/releases/download/${version}/${pname}-${version}.tar.gz";
-    sha256 = "1lbfhyyh9k54r8s1h8ca2czxv4hg0yq984kdh3vqh3990aca0x9a";
+    sha256 = "040d01pdzkj0nc1c0vsf6gfqf28cgil03ix8dasijvhiha4c20nz";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
   buildInputs = [
     tpm2-tss glib which dbus cmocka
```

Some derivation improvements :)

Is which required at runtime? If only at build-time, please put it in `nativeBuildInputs` or in `checkInput` if it's only needed for testing.

```suggestion
    tpm2-tss glib which
  ];
  
  checkInputs = [
    dbus cmocka
```